README.md

```diff
@@ -4,6 +4,22 @@ A webstore project built using the Spring Boot framework
 # Spring Boot framework Official Documentation
 https://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/html/
 
+# Cloning Webstore to IntelliJ on Windows
+
+IntelliJ Git setup:
+- To connect Git to IntelliJ enter `Ctrl Alt s` > `version control` > `Git`
+- Select the `...` in the `Path to Git executable` and enter the `git.exe` file
+- `git.exe` can be found in `Git` > `bin` 
+
+Cloning to IntelliJ:
+- From the main menu select `VCS` > `Checkout from version control` > `Git`
+- Enter Webstore git URL in the URL text box and assign a directory(If asked sign into your Github account)
```

Comment:
instead of saying `Enter Webstore git URL`, do you think it would be okay if we pasted the actual link to the git URL here? the link shouldn't ever change I think

`git@github.com:knutsun/webstore.git`

Reply:
Oh yes, that'd be better. It'd be more helpful to the others as well.